src/bin/pg_autoctl/keeper.c

```diff
@@ -587,13 +589,14 @@ keeper_get_replication_state(Keeper *keeper)
 				pgsql,
 				config->replication_slot_name,
 				postgres->pgsrSyncState,
-				&(postgres->walLag),
+				postgres->currentLSN,
+				NAMEDATALEN,
 				missing_wallag_ok);
+		log_warn("latest received lsn = %s", postgres->currentLSN);
```

Comment:
I think we should rename the function `pgsql_get_sync_state_and_wal_lag` because it's now getting the CurrentLSN rather than the WAL lag, and also rename our boolean `missing_wallag_ok` to be closer to what happens here (`missing_currentLSN_ok` I guess?).

Reply:
made it `missingStateOk`